index.html

```diff
@@ -2199,6 +2201,83 @@ <h2>
                 </li>
               </ol>
             </li>
+            <li>If <var>details</var>["<a>regionCode</a>"] is present and not
+            the empty string:
+              <ol>
+                <li>Let <var>regionPart</var> be the result of <a>strip leading
```

Comment:
The variable naming here is super-confusing. To a web developer, a regionCode is something like "CA", right? But in this algorithm, `details["regionCode"]` is "CA", `regionPart` is "CA", and `regionCode` is "USA-CA".

I think the variable on this line should be _regionCode_, and on line 2213 should be _putativeCountrySubdivisionCodeElement_ or similar.